k8s.go

```diff
@@ -0,0 +1,643 @@
+// Copyright (c) 2013 Kelsey Hightower
```

Comment:
It would be better to note (in the comment or the commit log) some part of this file is from confd/backends/k8s/client.go to ease maintenance.

Reply:
I will.

Reply:
Done.